src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -351,6 +351,414 @@ static S1 MayWrap(Span<int> arg)
         }
 
         [Fact()]
+        public void RefLikeScopeEscapeVararg()
+        {
+            var text = @"
+using System;
+class Program
+{
+    static void Main()
+    {
+        Span<int> outer = default;
+
+        S1 x = MayWrap(__arglist(outer));
+
+        {
+            Span<int> inner = stackalloc int[1];
+
+            // valid
+            x = MayWrap(__arglist(outer));
+
+            // error
+            x = MayWrap(__arglist(inner));
```

Comment:
Is the error here on the assignment or the call? Can the `__refvalue` of ref-like types be returned?

Reply:
Yes, it's value can be returned. 

---
In reply to: [141168290](https://github.com/dotnet/roslyn/pull/22303#discussion_r141168290) [](ancestors = 141168290)

Reply:
That would be like returning by value a by value parameter. Nothing is wrong with that on the callee side.

---
In reply to: [141169557](https://github.com/dotnet/roslyn/pull/22303#discussion_r141169557) [](ancestors = 141169557,141168290)